src/pages/docs/chat/api/javascript/message.mdx

```diff
@@ -0,0 +1,393 @@
+---
+title: Message
+meta_description: ""
+---
+
+The `Message` interface represents a single message in a chat room.
+
+---
+
+## Accessors
+
+It has the following accessors:
+
+| Accessor | Description | Type |
+| -------- | ----------- | ---- |
+| `deletedAt` | The timestamp at which the message was deleted. | Date \| `undefined` |
+| `deletedBy` | The clientId of the user who deleted the message. | String \| `undefined` |
```

Comment:
(IFF it was deleted)

Reply:
I've re-ordered them rather than added 'if' to everything. I think this is sufficient to have the Boolean first. 9c7b90d8a014b6e86a1508190809e427a40d7b08